test/test_torch.py

```diff
@@ -6208,19 +6208,25 @@ def test_comparison_ops(self):
     def test_comparison_ops_must_take_bool_output(self):
         with self.assertRaisesRegex(RuntimeError, 'The output tensor of lt must be a bool'):
             torch.lt(torch.tensor([True]), torch.tensor([False]), out=torch.empty(1, dtype=torch.uint8))
+            torch.le(torch.tensor([True]), torch.tensor([False]), out=torch.empty(1, dtype=torch.uint8))
```

Comment:
We can abstract away the operation being called so we don't have to list one line per operation. Something like:
```
ops = [torch.lt, torch.le, ...]
for op in ops:
    with self.assertRaisesRegex(...)
```